bot_telegram/starter.py

```diff
@@ -0,0 +1,51 @@
+from tornado import gen, httpclient, ioloop
+import json
+from bot_telegram.telezombie import api
+
+
+class KelThuzad(api.TeleLich):
+
+    def __init__(self, api_token, ws_url):
+        super().__init__(api_token)
+        self.ws_url = ws_url
+
+    @gen.coroutine
+    def on_text(self, message):
+        id_ = message.message_id
+        chat = message.chat
+        text = message.text.strip().replace(" ", "-").replace("[()&?]", "")
+        if not self.is_ascii(text):
```

Comment:
Why not just search for the text and print smth like "No results matching the request" ? Is there some restrictions on the search text?